.github/workflows/test.yml

```diff
@@ -27,14 +27,15 @@ jobs:
         (New-Object System.Net.WebClient).DownloadFile("http://prdownloads.sourceforge.net/swig/swigwin-4.0.1.zip","swigwin-4.0.1.zip");
         Expand-Archive .\swigwin-4.0.1.zip .;
         echo "$((Get-Item .).FullName)/swigwin-4.0.1" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
-    - name: Install APT dependencies
-      if: "startsWith(runner.os, 'ubuntu')"
-      run: |
-        sudo apt-get install -y git build-essential apt-utils wget libfreetype6 libpng-dev libopenblas-dev gcc gfortran libsnappy-dev
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip wheel
         pip install . -r requirements.txt
+        pip install -e .
+    - name: Run eegnb
+      run: |
+        eegnb --help
+        eegnb runexp
```

Comment:
This is where the 'fun' errors start popping up:
Two of them (Mac + Windows) seem related to uninstalled dependencies (h5py), which might indicate an issue similar to: https://github.com/NeuroTechX/eeg-notebooks/pull/26#issuecomment-738545393

The other error is on Linux, and that might be due to the runtime environment not having X-Windows or some type of display mechanism (not sure though)

Github Action Test Failures: https://github.com/NeuroTechX/eeg-notebooks/runs/1501111227

Windows: 
```
 0.3177 	WARNING 	Monitor specification not found. Creating a temporary one...
  File "C:\hostedtoolcache\windows\Python\3.6.8\x64\Scripts\eegnb-script.py", line 11, in <module>
11.5925 	WARNING 	We strongly recommend you activate the PTB sound engine in PsychoPy prefs as the preferred audio engine. Its timing is vastly superior. Your prefs are currently set to use ['sounddevice', 'PTB', 'pyo', 'pygame'] (in that order).
    load_entry_point('eeg-notebooks', 'console_scripts', 'eegnb')()
  File "d:\a\eeg-notebooks\eeg-notebooks\eegnb\cli\__main__.py", line 73, in main
    cli = CLI(args.command)
  File "d:\a\eeg-notebooks\eeg-notebooks\eegnb\cli\cli.py", line 9, in __init__
    getattr(self, command)()
  File "d:\a\eeg-notebooks\eeg-notebooks\eegnb\cli\cli.py", line 41, in runexp
    from .utils import run_experiment
  File "d:\a\eeg-notebooks\eeg-notebooks\eegnb\cli\utils.py", line 10, in <module>
    import h5py
ModuleNotFoundError: No module named 'h5py'
Error: Process completed with exit code 1.
```

Mac

```
 2.9720 	WARNING 	Monitor specification not found. Creating a temporary one...
  File "/Users/runner/hostedtoolcache/Python/3.6.12/x64/bin/eegnb", line 11, in <module>
5.3298 	WARNING 	We strongly recommend you activate the PTB sound engine in PsychoPy prefs as the preferred audio engine. Its timing is vastly superior. Your prefs are currently set to use ['sounddevice', 'PTB', 'pyo', 'pygame'] (in that order).
    load_entry_point('eeg-notebooks', 'console_scripts', 'eegnb')()
  File "/Users/runner/work/eeg-notebooks/eeg-notebooks/eegnb/cli/__main__.py", line 73, in main
    cli = CLI(args.command)
  File "/Users/runner/work/eeg-notebooks/eeg-notebooks/eegnb/cli/cli.py", line 9, in __init__
    getattr(self, command)()
  File "/Users/runner/work/eeg-notebooks/eeg-notebooks/eegnb/cli/cli.py", line 41, in runexp
    from .utils import run_experiment
  File "/Users/runner/work/eeg-notebooks/eeg-notebooks/eegnb/cli/utils.py", line 10, in <module>
    import h5py
ModuleNotFoundError: No module named 'h5py'
Unable to create basic Accelerated OpenGL renderer.
Unable to create basic Accelerated OpenGL renderer.
Core Image is now using the software OpenGL renderer. This will be slow.
Error: Process completed with exit code 1.
```

Linux (looks like this failure is mostly due to a lack of display in the build system):
```
    load_entry_point('eeg-notebooks', 'console_scripts', 'eegnb')()
  File "/home/runner/work/eeg-notebooks/eeg-notebooks/eegnb/cli/__main__.py", line 73, in main
    cli = CLI(args.command)
  File "/home/runner/work/eeg-notebooks/eeg-notebooks/eegnb/cli/cli.py", line 9, in __init__
    getattr(self, command)()
  File "/home/runner/work/eeg-notebooks/eeg-notebooks/eegnb/cli/cli.py", line 41, in runexp
    from .utils import run_experiment
  File "/home/runner/work/eeg-notebooks/eeg-notebooks/eegnb/cli/utils.py", line 2, in <module>
    from eegnb.experiments.visual_n170 import n170
  File "/home/runner/work/eeg-notebooks/eeg-notebooks/eegnb/experiments/visual_n170/n170.py", line 9, in <module>
    from psychopy import visual, core, event
  File "/opt/hostedtoolcache/Python/3.6.12/x64/lib/python3.6/site-packages/psychopy/visual/__init__.py", line 28, in <module>
    from .button import ButtonStim
  File "/opt/hostedtoolcache/Python/3.6.12/x64/lib/python3.6/site-packages/psychopy/visual/button.py", line 12, in <module>
    from psychopy import event
  File "/opt/hostedtoolcache/Python/3.6.12/x64/lib/python3.6/site-packages/psychopy/event.py", line 61, in <module>
    _default_display_ = pyglet.canvas.get_display()
  File "/opt/hostedtoolcache/Python/3.6.12/x64/lib/python3.6/site-packages/pyglet/canvas/__init__.py", line 94, in get_display
    return Display()
  File "/opt/hostedtoolcache/Python/3.6.12/x64/lib/python3.6/site-packages/pyglet/canvas/xlib.py", line 123, in __init__
    raise NoSuchDisplayException('Cannot connect to "%s"' % name)
pyglet.canvas.xlib.NoSuchDisplayException: Cannot connect to "None"
Error: Process completed with exit code 1.
```